arch/x86/chorba_sse2.c

```diff
@@ -0,0 +1,853 @@
+#if !defined(WITHOUT_CHORBA) && (defined(X86_SSE2) || defined(X86_NOCHECK_SSE2))
+
+#include "zbuild.h"
+#include "crc32_braid_p.h"
+#include "crc32_braid_tbl.h"
+#include "crc32.h"
+#include <emmintrin.h>
+#include <immintrin.h>
+#include <smmintrin.h>
```

Comment:
Is it necessary to have the headers for later SSE versions?

Reply:
Sorry, was trying to figure out what was going on with cvtsi64x_si128 et al.  Evidently they just remove them from 32 bit compilations to make our lives more difficult.